src/scenes/NurseryRouter/PlantingProgressTabContent.tsx

```diff
@@ -55,29 +99,17 @@ export default function PlantingProgress({ reloadTracking }: PlantingProgressPro
       onSearch: (value: string) => setSearch(value),
       filtersProps: {
         filters,
-        setFilters: (value: Record<string, any>) => setFilters(value),
+        setFilters,
         filterColumns,
         filterOptions,
       },
     }),
-    [filters, filterColumns, filterOptions, search]
+    [search, filters, filterColumns, filterOptions]
```

Comment:
search at the end ?

Reply:
You bet... I gotta look into why my auto-add-dependencies functionality in IntelliJ keeps moving these things around...